tiatoolbox/annotation/storage.py

```diff
@@ -2205,14 +2284,18 @@ def _query(
 
         """
         if not no_constraints_ok and all(x is None for x in (geometry, where)):
-            raise ValueError("At least one of `geometry` or `where` must be specified.")
+            msg = "At least one of `geometry` or `where` must be specified."
+            raise ValueError(msg)
         query_geometry = geometry
         if callable_columns is None:
             callable_columns = columns
         if geometry_predicate not in self._geometry_predicate_names:
+            msg = (
+                f"Invalid geometry predicate.Allowed values are: "
```

Comment:
no f-string, add space

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/991322d859c0c22e7c7a1d713362944aa4317198